### PR TITLE
Promote the Cubano framework from the main Concordion site

### PR DESCRIPTION
**Pull request by nigelcharman**

### CODE REVIEW

**Comment by nigelcharman** [comment]
The Jenkins project is showing a 403 on https://wiki.jenkins.io/display/JENKINS/Configuring+Content+Security+Policy. I'll report to them tomorrow.

File: _includes/extensions.md [hunk 1]
```diff
@@ -359,6 +359,8 @@ For inspiration, have a look at the [Java-based extensions]({{ site.baseurl }}/{
 </div>
 </section>
 
+See also the [Cubano]({{site.baseurl}}/cubano) framework which integrates Concordion with a number of Concordion extensions, WebDriver and other open-source projects to provide a ready-made framework for web and API testing.
```
Comment (andrew-sumner) [inline _includes/extensions.md]: Should we use "Selenium WebDriver" rather than WebDriver?

File: _includes/questions.md [hunk 2]
```diff
@@ -37,9 +37,9 @@ Concordion integrates well with [Selenium WebDriver](http://www.seleniumhq.org/p
...
+To see the range of options available for browser management, download the [concordion-scope-examples](https://github.com/concordion/concordion-scope-examples/blob/master/README.md) project. This provides sample code for each permutation of browser scope, and parallel or serial testing. It runs tests against the Google home page and generates specifications that show a screenshot on failure (see the Arithmetic specification). You might also want to add other [extensions]({{site.baseurl}}/extensions/{{ page.fixture_language }}/{{ page.spec_type }}), such as Storyboard, Logback or others.
 
-You might also want to add other [extensions]({{site.baseurl}}/extensions/{{ page.fixture_language }}/{{ page.spec_type }}), such as Storyboard, Logback or others.
+The [Cubano]({{site.baseurl}}/cubano) framework integrates Concordion with a number of Concordion extensions, WebDriver and other open-source projects to provide a ready-made framework for web and API testing that creates beautiful living documentation.
```
Comment (andrew-sumner) [inline _includes/questions.md]: Should we use "Selenium WebDriver" rather than WebDriver?

**Review by jimmykemp — APPROVED** [review]
Think this is a good start.  Perhaps down the track we could move the Cubano info, up near the top of the extensions page.  Or promote from outside one of these three entry points

**Pull request merged**